src/NucleiEnv.php

```diff
@@ -64,37 +61,33 @@ public function __construct(
      */
     public function prepareCommand(
         string $spEntityId,
-        string $target,
         string $ascUrl,
         string $token,
         string $testId = null,
     ): string {
         $spTestResultsDir = $this->getSpTestResultsDir($spEntityId);
-        $fileName = $this->helpers->filesystem()->cleanFilename($spEntityId);
+        $screenshotsDir = escapeshellarg(rtrim($spTestResultsDir . '/') . '/pictures/');
 
         // Escape shell args.
         $spEntityId = escapeshellarg($spEntityId);
         $target = escapeshellarg($target);
         $acsUrl = escapeshellarg($ascUrl);
-        $token = escapeshellarg($token);
+        $authorization = escapeshellarg("Bearer $token");
```

Comment:
I have actually enabled token to be provided not only in authorization header, but also in HTTP body (not documented feature currently). This would hardcode it to Bearer type only...

Reply:
Actually nuclei added a generic authentication system a couple of days ago, so I will switch to that. Therefore we will not hardcode anything.